src/asciidoc/reference/packages/withkojiCustomVccSdk.adoc

```diff
@@ -176,6 +168,58 @@ customVcc.uploadFile(myBlob, myFileName, (url) => {
 });
 ----
 
+== CustomVcc objects
+
+=== Props
+
+A `props` object respresents the properties representing the current state of a custom VCC.
+It is returned to the handler listening to <<onUpdate>>.
+
+The `props` object includes the following properties.
+
+[source,javascript]
+----
+{
+	type: string;<1>
+	name: string;<2>
+	value: any;<3>
+	scope: string;<4>
+	variableName: string;<5>
+	options: object;<6>
+	collaborationDecoration: object;<7>
+	_config: object;<8>
+};
+----
+<1> `type` – the type signature for this vcc.
+<2> `name` – string name of the VCC.
+<3> `value` – current value of the VCC.
+<4> `scope` – name of the section where this vcc appears.
+<5> `variableName` – resolved variable name of this vcc (`scope.key`).
+<6> `options` – an object containing any options passed in `typeOptions`.
+<7> `collaborationDecoration` – an object containing any collaborators currently focused on this control.
+<8> `_config` – the full VCC configuration file. Generally this is not needed, as most controls should be fully isolated to a single value, but this can be useful when creating more complex custom controls like map builders.
+
+=== Theme
+
+A `theme` object allows you to style using matching colours and styles from remixers' active theme.
+It is returned to the handler listening to <<onTheme>>.
+
+The `theme` object includes the following properties.
+
+[source,javascript]
+----
+{
+	name: string;<1>
+	breakpoints: object;<2>
+	colors: object;<3>
+	mixins: object;<4>
+};
+----
+<1> `name` – name of the theme.
+<2> `breakpoints` – an object contain responsive style breakpoints of the theme.
```

Comment:
```suggestion
<2> `breakpoints` – An object contain responsive style breakpoints of the theme.
```
It would be great to get an example of the properties in this object (also colors and mixins). Maybe we can just add a sample of a returned `theme` object value below. Alternatively, call out a few key properties in each object as examples.